docs/index.rst

```diff
@@ -12,6 +12,19 @@ Contents:
    api-reference/index
 
 
+Warehouse is a new code base that implements a Python package index. It is being actively developed and the plan is that it will eventually power https://pypi.python.org/pypi (PyPI) and replace an older code base that is currently powering PyPI. You can see Warehouse in production at https://warehouse.python.org
```

Comment:
Also I'd remove the explicit links here to PyPI and link them like:

it will eventually power [PyPI](https://pypi.python.org/), ideally omitting the trailing ``/pypi`` from the link as well.